### PR TITLE
Add pullrequest icon

### PR DESCRIPTION
**Pull request by nomean42**

### CODE REVIEW

**Comment by knownasilya** [comment]
Where is this supposed to show up?

File: client/job-status/controllers/job.js [hunk 1]
```diff
@@ -108,6 +108,10 @@ module.exports = function ($scope, $route, $location, $filter) {
   });
 
   $scope.triggers = {
+    pullrequest: {
+      icon: 'share-square-o',
+      title: 'Pullrequest'
```
Comment (knownasilya) [inline client/job-status/controllers/job.js]: Should be `Pull request`

**Comment by knownasilya** [comment]
Ping

**Comment by nomean42** [comment]
Hi
i am trying to make bitbucket pull request handling,  you can check it in
my git repo
https://github.com/nomean42/strider-bitbucket/tree/feature/add-pullrequest-handling

On Sat, Jan 30, 2016 at 7:17 AM, Ilya Radchenko <notifications@github.com>
wrote:

> Ping
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Strider-CD/strider/pull/905#issuecomment-177038957>.
>



-- 
С уважением, Горяйнов Игорь.

File: client/job-status/controllers/job.js (hunk 1 above)
Comment (nomean42) [inline client/job-status/controllers/job.js]: Quite agree, but i cant find better one... Any suggestions?

**Comment by knownasilya** [comment]
Feel free to open the bitbucket PR and just keep commiting there so we can see your progress.

**Pull request closed without merging**